### DISCUSSION TITLE
[Feature Request] Backup to WebDav

### DISCUSSION THREAD

**Discussion by MorgothSauron** [standardnotes/forum#2613] [Suggestions]
I currently receive backups using dropbox. As part of my plan to improve my privacy on the internet I'm planning to stop using cloud services from dropbox, microsoft or google.

I know the backups are already encrypted. However it would be nice to propose backup using webdav (e.g. nextcloud).

Is there any plan to integrate support of WebDav as a backup destination ?

**Comment by JaspalSuri** [comment]
Hello fellow Middle-Earthian,

I'm afraid we can't comment on our current plans, but I will pass your feedback along to the team!

**Comment by atmoio** [comment]
We're definitely tracking this internally, and I agree it would be good to have. It's just impossible to say when we'll get a chance to build this out.

**Comment by cfl0ws** [comment]
I'd also really like to see this feature implemented. Given Standard Notes focus on privacy, it would seem natural to support backing up to self-owned data storage locations and not be limited to "The Big Three" 😬 🙂
